build/buildconfig.yaml

```diff
@@ -31,5 +30,6 @@ spec:
     type: Git
     git:
       uri: 'https://github.com/bcgov/appassessment'
+      ref: 'UI/UX-improvements-for-the-Assessment-App'
```

Comment:
Do you want to test build image off from this branch? otherwise, it should stay as main.
The way I usually do is manually update ref in openshift to the test branch that I want image build from

Reply:
Yup, this isn't going into main. OK, I'll start doing what you said here.